msisdn_cli/__init__.py

```diff
@@ -39,6 +39,7 @@
   --mnc=<mnc>            Mobile Network Code (2 or 3 digits) ie: 07
   -n --msisdn=<msisdn>   The MSISDN number you want to validate.
   -v, --verbose          Display the assertion
+  --no-ssl-verify        Allow wrong SSL configuration
```

Comment:
Maybe (*just an idea*), we could reuse the option name and description of other commands

like wget
```
$ wget --help
...
--no-check-certificate   don't validate the server's certificate.
```
or curl
```
$ curl --help
...
 -k, --insecure      Allow connections to SSL sites without certs (H)
```